controller/src/sensors.cpp

```diff
@@ -1,24 +1,112 @@
-/* Copyright 2020, Edwin Chiu
+/* Copyright 2020, RespiraWorks
 
-  This file is part of FixMoreLungs.
+  This file is part of VentilatorSoftware.
 
-  FixMoreLungs is free software: you can redistribute it and/or modify
+  VentilatorSoftware is free software: you can redistribute it and/or modify
   it under the terms of the GNU General Public License as published by
   the Free Software Foundation, either version 3 of the License, or
   (at your option) any later version.
 
-  FixMoreLungs is distributed in the hope that it will be useful,
+  VentilatorSoftware is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.
 
   You should have received a copy of the GNU General Public License
-  along with FixMoreLungs.  If not, see <https://www.gnu.org/licenses/>.
+  along with VentilatorSoftware.  If not, see <https://www.gnu.org/licenses/>.
 */
+
+/*
+ module contributors: verityRF, lee-matthews, Edwin Chiu
+ The purpose of this module is to allow calibrated readings from the different pressure sensors in the ventilator design. 
+ It is designed to be used with the Arduino Nano and the MPXV5004GP and MPXV7002DP pressure sensors.
+*/
+
+#ifndef _TEST_
 #include <Arduino.h>
+#else
+#include "ArduinoSim.h"
+#endif
```

Comment:
This will be a bummer to put everywhere.

Can we instead have a header which does this?

Reply:
There might be a way around it, but this is the simplest way I could think of. When running the unit tests we really can't have Arduino.h being referenced since that means nothing to the host running the tests. There might be a way to get the actual Arduino.h path to change between the real header and our fake one using some project configuration trickery. Though this method at least keeps the two very visibly different, so you don't end up trying to put the fake one on an actual Arduino.

Reply:
What if we made a header called `Arduino.h` which did this and then

 - Did `#include "Arduino.h` (note not `<Arduino.h>`)
 - Changed everything to the quotes version.
 - Wrote a http://pre-commit.com to prevent us from using the angle-brackets version?

If you want I can do that as a separate change, the last thing I want to do is slow anyone down.

Reply:
On a loosely related note, we could have `#pragma once` for header safeguards. Wonder if those work for arduino compiler?

Reply:
re `pragma once`, we can use it and it's probably fine, but here's the reason that most projects I've seen avoid it: https://stackoverflow.com/a/47830826

Reply:
we use #pragma once at work and it's been fine, but our codebase is relatively small

Reply:
Could try this as a separate change I think, we can follow up on it later.